client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.controller.js

```diff
@@ -226,5 +216,9 @@ angular.module("App").controller(
 
             this.shouldDisplayCheckMigrationsErrors = true;
         }
+
+        cancel () {
+            console.log("cancel !", this);
```

Comment:
-`console` :)

Reply:
Oops, I forgot removing this test !
Done